radicale/storage.py

```diff
@@ -207,19 +237,42 @@ def __init__(self, path):
 
 
 class Item:
-    def __init__(self, collection, item, href, last_modified=None):
+    def __init__(self, collection, href, last_modified=None,
+                 text=None, item=None, etag=None):
```

Comment:
> Please add a doc-string

Yes.

> The interface changed.

Totally missed that. I also changed the behavior of ``pre_filtered_list``.

> This should go into the release notes.

Or I reorder the parameters and make it backwards compatible. It depends, if this PR gets merged before the 2.0.0 release.